templates/firefox-ios.xliff

```diff
@@ -3707,6 +3731,279 @@ Private Tab</source>
+        <source>Can’t Check These Reviews</source>
+        <note>Title for info card when Fakespot cannot analyze reviews for a certain product type</note>
+      </trans-unit>
+      <trans-unit id="Shopping.InfoCard.NeedsAnalysis.PrimaryAction.v120" xml:space="preserve">
+        <source>Check Now</source>
+        <note>Primary action title for info card when the product needs analysis</note>
+      </trans-unit>
+      <trans-unit id="Shopping.InfoCard.NeedsAnalysis.Title.v120" xml:space="preserve">
+        <source>New Info To Check</source>
+        <note>Title for info card when the product needs analysis</note>
+      </trans-unit>
+      <trans-unit id="Shopping.InfoCard.NoInfoAvailableRightNow.Description.v120" xml:space="preserve">
+        <source>We’re working to resolve this issue. Please check back soon.</source>
+        <note>Description for info card when no information is available at the moment</note>
+      </trans-unit>
+      <trans-unit id="Shopping.InfoCard.NoInfoAvailableRightNow.Title.v120" xml:space="preserve">
+        <source>No Info Available Right Now</source>
+        <note>Title for info card when no information is available at the moment</note>
+      </trans-unit>
+      <trans-unit id="Shopping.InfoCard.NotEnoughReviews.Description.v120" xml:space="preserve">
+        <source>When this product has more reviews, we’ll be able to check their quality.</source>
+        <note>Description for info card when there are not enough reviews for a product</note>
+      </trans-unit>
+      <trans-unit id="Shopping.InfoCard.NotEnoughReviews.Title.v120" xml:space="preserve">
+        <source>Not Enough Reviews Yet</source>
+        <note>Title for info card when there are not enough reviews for a product</note>
+      </trans-unit>
+      <trans-unit id="Shopping.InfoCard.ProgressAnalysis.Description.v120" xml:space="preserve">
+        <source>This could take about 60 seconds.</source>
+        <note>Description for info card when the product is in analysis mode</note>
+      </trans-unit>
+      <trans-unit id="Shopping.InfoCard.ProgressAnalysis.Title.v120" xml:space="preserve">
+        <source>Checking review quality</source>
+        <note>Title for info card when the product is in analysis mode</note>
+      </trans-unit>
+      <trans-unit id="Shopping.NoAnalysisCard.AnalyzerButton.Title.v120" xml:space="preserve">
+        <source>Check Review Quality</source>
+        <note>Text for the analyzer button, from No Analysis Card View displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.NoAnalysisCard.BodyLabel.Title.v120" xml:space="preserve">
+        <source>To know whether this product’s reviews are reliable, check the review quality. It only takes about 60 seconds.</source>
+        <note>Text for the body label, from No Analysis Card View displayed in the shopping review quality bottom sheet.</note>
```

Comment:
"from No Analysis Card View displayed in the shopping review quality bottom sheet"
-> also a bit confusing for localizers. 
Could be just "text for body label, to check reliability of product"

Reply:
Fixed in https://github.com/mozilla-mobile/firefox-ios/pull/16940